pkg/webhooks/subscription/subscription.go

```diff
@@ -0,0 +1,182 @@
+package subscription
+
+import (
+	"encoding/json"
+	"net/http"
+	"sync"
+
+	responsehelper "github.com/openshift/managed-cluster-validating-webhooks/pkg/helpers"
+	"github.com/openshift/managed-cluster-validating-webhooks/pkg/webhooks/utils"
+	"k8s.io/api/admission/v1beta1"
+	admissionregv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	admissionctl "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	WebhookName    string = "subscription-validation"
+	LoggingSubName string = "cluster-logging"
+	ESSubName      string = "elasticsearch-operator"
+	Channel        string = "4.5"
```

Comment:
What does `Channel` signify? Is it a channel that's blocked somehow?

Reply:
Yes, 4.5 is the channel we want to block from being installed or upgraded to

Reply:
May I suggest changing it to `BlockedChannel` or something like that?

Reply:
+1